samples/maps/demo/blend-colors/demo.js

```diff
@@ -4,7 +4,11 @@
         'https://code.highcharts.com/mapdata/custom/world.topo.json'
     ).then(response => response.json());
 
-    Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@v7.0.0/samples/data/world-population.json', function (data) {
+    Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@b0488c7087/samples/data/busiest-europe-airports-2021.json', data => {
```

Comment:
Will the final URL look like this?

Reply:
I did it for the first time in my life, so I am also not sure if that is correct, I was based on: https://github.com/highcharts/highcharts/tree/master/samples/data#sample-data-folde

Reply:
It is okay, we need to do it that way for new demo content. Later, when the next release is tagged, we can (optionally) change it.